### DISCUSSION TITLE
Can't find the BASE_DIR

### DISCUSSION THREAD

**Discussion by zurtri** [django-commons/django-unicorn#213] [Q&A] [answered]
Hi guys,

Just starting and trying to create my first component I get this error:

`python manage.py startunicorn hello_world
CommandError: Can't find BASE_DIR for this project`

I have this in my settings.py:
`    'django_unicorn',
    # 'unicorn',`

If I uncomment unicorn I get:
`ModuleNotFoundError: No module named 'unicorn'`

Any ideas on how I can get it finding the BASE_DIR?

Thanks!

**Comment by zurtri** [comment, marked as answer]
Found the solution!

Just had to put `BASE_DIR = APPS_DIR` in my setting.py file.

I am using Django CookieCutter so that is probably where the misconfig was.

**Reply by adamghill** [reply]
Awesome that you figured out what was going on! I made an issue to hopefully fix this for others in the future: https://github.com/adamghill/django-unicorn/issues/214.

**Reply by zurtri** [reply]
Nice - thanks mate!

**Reply by adamghill** [reply]
This has been fixed in https://github.com/adamghill/django-unicorn/releases/tag/0.30.0.
